cmd/panda_test_executor/panda_executor_test.go

```diff
@@ -0,0 +1,456 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"regexp"
+	"strconv"
+	"strings"
+	"testing"
+
+	controller "github.com/panda-re/panda_studio/internal/panda_controller"
+)
+
+// Each enum represents the state panda was in that caused the exception
```

Comment:
We should put this in another file somewhere, a readme that goes over tests, errors and what each one that we have encountered means

Reply:
Added a readme with that info